changelog.txt

```diff
@@ -39,6 +40,7 @@ Fixed
 Deprecated
 ++++++++++
 
+ - Deprecate the get_id method in ``Project`` and ``Job`` classes in favor of the id property, to be removed in 2.0 (#250)
```

Comment:
```suggestion
 - The ``get_id`` method in ``Project`` and ``Job`` classes in favor of the ``id`` property, to be removed in 2.0 (#250)
```

Reply:
@csadorf you commented that you would prefer full sentences. The suggested change makes the sentence a fragment. I don't mind, but is that what you prefer?

Reply:
Sorry, I responded to an outdated version of the code. All I wanted to suggest were the added code markups.